decrypt/decrypt.go

```diff
@@ -25,7 +25,7 @@ var privateKey = Args.String("key", "",
 	"Private key to use for decrypting files.")
 
 // Main decryption function
-func Decrypt(args []string) {
+func Decrypt(args []string) error {
```

Comment:
Should this change be here? it is not related to encryption

Reply:
it's not. but it's a few lines to make all the modules behave the same, so I figured I could just include it. would you prefer if I break it out into a separate PR?